frontend/WikiContrib-Frontend/src/query.js

```diff
@@ -414,6 +465,42 @@ export class Query extends Component {
         ) : (
           <React.Fragment>
             <NavBar />
+            <Modal
+              id="fuzzy-warning"
+              open={!this.state.showMismatch}
+              onClose={()=> this.setState({showMismatch:true,loading:false})}
+              size='tiny'
+              >
+              <Modal.Header>Warning!
+                <Icon
```

Comment:
This icon, I think we can either remove or modify such that hovering on it changes the pointer to a hand cursor. I'm inclined for just removing it as we have a "No" button that does the same. Whichever is easy implementation wise.

Reply:
I think removing is better. It is always better to reduce code when possible than add to it